### DISCUSSION TITLE
Self Pooling - Rewards are getting less and less frequent

### DISCUSSION THREAD

**Discussion by hcm609** [Chia-Network/chia-blockchain#9281] [Q&A]
Hi.
I am self Pooling and I have experienced that the XCH I get are not so much like before.
The time to win a block is at ~ 15 hours and it's been 4 days and I didnt get a dime!!!
I know it's a estimation and not a contract but it's very awkward to be zero?
Does chia has issues or just all farmers are abandoning the project?

**Comment by mdrodge** [comment]
By self pooling, do you mean solo farming?
Are you getting consistent look ups? (not farming via SMB etc?)
Is your routers port forwarding set up correctly?

Re***   Does chia has issues or just all farmers are abandoning the project?     Chia is fine and if other farmers leave that will only increase your odds of getting blocks.

Describe your set up a little.
SAS Adaptor? Raid? OS? RAM? 
So far all we have is the fact that your unhappy and nothing to help you with.

**Reply by hcm609** [reply]
I do solo farming.
I have a space of 10PiB with one main full node and the rest are in storage machines (SAS) with a harvester in each of them.
I do get frequent lookups every 10s.
No errors in any log file.
However I do not have frequent rewards and I was wondering why since in the past it was higher rewards.
Any ideas?

**Reply by rmica** [reply]
Are you on NFT self pooling or OG?

**Comment by mdrodge** [comment]
Well odds play into it. For example your previous might have been
statistically higher than the normal.
Your current might be statistically lower than the normal.
So the difference might seem more than you expected but it's still a
function of random numbers.
Were you effected by the dust storm at all? Do you have plenty of
connections up to date peers.

**Reply by hcm609** [reply]
Ok I will let sometime to pass and see on the gains.
I was not affected by the dust storm since the servers were colocated in a data center.

**Comment by mdrodge** [comment]
Since so many were effected and this is a network of shares i would say
that you probably were effected.
The degree at which you were effected may vary but that might be difficult
to track now. It's definitely not as simple as assuming being in a
datacentre would offer some protection. If all your peers weren't keeping
up you would have also slipped behind.
The one way to tell that would be to look at your exact share times. If you
were effected you'd see a stacking of times (100 lookups for minute x and 0
in minutes y and z)

**Reply by rmica** [reply]
I noticed both the stacking of times during the storm and less frequent rewards. I'm checking the farm twice a day. Lookups are now almost normal (29-34 in 5m range), and the only bad news that I read in the logs are 3-4 block validation time warnings a day (only slightly above 2 secs).
